recipes/vacumm/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "vacumm" %}
+{% set version = "3.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7107752876047704bd122524bc4a95eaf48ce7bcda6ea8a6e0cda81d1e6be654
+
+build:
+  number: 0
+  #script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win or py3k or np<111]
```

Comment:
`np<111` is skipping all the builds. If the package only works with numpy 1.11+, you should remove this and use `numpy 1.11.*` in the `host` requirements.

Reply:
Ok thanks. 
Is the same hold for python 2.7?

Reply:
In fact, it work for numpy 1.11 and later. So ``numpy >=1.11``?